### PR TITLE
MCOL-4011 update custom loadbrm script

### PR DESCRIPTION
**Pull request by jmrojas2332**
Updates based on feedback for original MCOL-4011 PR https://github.com/mariadb-corporation/mariadb-columnstore-engine/pull/1233

### CODE REVIEW

File: oam/install_scripts/mcs-loadbrm.py [hunk 1]
```diff
@@ -2,17 +2,19 @@
...
 cs_config = ET.parse('/etc/columnstore/Columnstore.xml')
 config_root = cs_config.getroot()
 
-storage = sm_config['ObjectStorage']['service']
-region = sm_config['S3']['region']
-bucket = sm_config['S3']['bucket']
-loadbrm = '/usr/bin/load_brm /var/lib/columnstore/data1/systemFiles/dbrm/{0}'
+storage = sm_config.get('ObjectStorage', 'service', fallback='LocalStorage')
+region = sm_config.get('S3', 'region', fallback='some_region')
+bucket = sm_config.get('S3', 'bucket', fallback='some_bucket')
```
Comment (jmrojas2332) [inline oam/install_scripts/mcs-loadbrm.py:15]: Read with a default

File: oam/install_scripts/mcs-loadbrm.py [hunk 2]
```diff
@@ -2,17 +2,19 @@
...
+dbrmroot = config_root.find('./SystemConfig/DBRMRoot').text
```
Comment (jmrojas2332) [inline oam/install_scripts/mcs-loadbrm.py:16]: Getting dbrmroot dynamically. Note: didn't switch to using lxml as i don't think adding a new dependency is a good idea this late in the game

File: oam/install_scripts/mcs-loadbrm.py [hunk 3]
```diff
@@ -46,17 +48,17 @@
...
         try:
             brm_saves_current = subprocess.check_output(['cat', brm])
         except subprocess.CalledProcessError as e:
             # will happen when brm file does not exist
-            pass
+            print(f'{brm} does not exist.', file=sys.stderr)
 
 if brm_saves_current:
-    cmd = loadbrm.format(brm_saves_current.decode('utf-8'))
+    cmd = f'{loadbrm} {dbrmroot}{brm_saves_current.decode("utf-8").replace("BRM_saves", "")}'
```
Comment (jmrojas2332) [inline oam/install_scripts/mcs-loadbrm.py:61]: Both dbrmroot and brm_saves_current variable may contain BRM_saves string so let's replace it in brm_saves_current

File: oam/install_scripts/mcs-loadbrm.py (hunk 2 above)
Comment (drrtuy) [reply oam/install_scripts/mcs-loadbrm.py:16]: OK, let's do this after the release.

**Review by drrtuy — APPROVED** [review]

**Pull request merged**